zgrab2_schemas/zgrab2/smb.py

```diff
@@ -62,7 +65,7 @@ def extended(base, new):
             "smb_encryption_support": Boolean(doc="Server supports encryption"),
             }, doc="Capabilities flags for the connection. See [MS-SMB2] Sect. 2.2.4."),
         'negotiation_log': negotiate_log,
-        'has_ntlm': Boolean(),
+        'has_ntlm': Boolean(doc="Server supports the NTLM authentication method"),
```

Comment:
There _will_ be cases where `ntlm` is populated but `has_ntlm` is false, so some clarification is needed